net.davidotek.pupgui2.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "net.davidotek.pupgui2",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.llvm12"
+    ],
+    "command": "net.davidotek.pupgui2",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--persist=.config/pupgui",
```

Comment:
This shouldn't be needed, as long as your application respects $XDG_CONFIG_HOME variable.